app/src/main/java/org/p2p/wallet/rpc/repository/balance/RpcBalanceInteractor.kt

```diff
@@ -0,0 +1,28 @@
+package org.p2p.wallet.rpc.repository.balance
+
+import org.koin.ext.getFullName
+import org.p2p.wallet.rpc.repository.amount.RpcAmountRepository
+import timber.log.Timber
+
+class RpcBalanceInteractor(
```

Comment:
Imho, this is about amount. No need to create balance interactor and all other repositories.
Let's have one repository for these three cases: 
fees, minBalanceForRent, and lamportsPerSignature